nextflow_schema.json

```diff
@@ -273,6 +273,12 @@
             "default": "",
             "fa_icon": "fas fa-filter",
             "properties": {
+                "skip_variant_calling": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Skip variant calling for all samples.",
+                    "fa_icon": "fas fa-toggle-off"
+                },
```

Comment:
Perhaps have these together with the other skips below?

Reply:
Done! Thanks for the super fast review 😄